grabbit/src/main/resources/META-INF/spring/client-batch-job.xml

```diff
@@ -33,12 +33,18 @@
 
     <import resource="client-batch-job-context.xml"/>
 
+    <import resource="client-validateJob.xml" />
     <import resource="client-namespace-step.xml" />
     <import resource="client-jcrNodes-step.xml" />
     <import resource="client-deleteBeforeWrite-steps.xml" />
 
     <batch:job id="clientJob" xmlns="http://www.springframework.org/schema/batch" job-repository="clientJobRepository">
 
+        <batch:decision id="validateJob" decider="validJobDecider">
+            <next on="VALID" to="clientWorkflowOff" />
+            <fail on="INVALID" exit-code="VALIDATION_FAILED" />
+        </batch:decision>
+
```

Comment:
However it looks like the `deleteBeforeWriteDecider` is getting hit correctly. So I am not sure why the `validJobDecider` is NOT getting hit ....